staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -39,21 +39,23 @@ const (
 	EvictionKind = "Eviction"
 	// EvictionSubresource represents the kind of evictions object as pod's subresource
 	EvictionSubresource = "pods/eviction"
+	podSkipMsgTemplate  = "pod %q has DeletionTimestamp older than %v seconds, skipping\n"
 )
 
 // Helper contains the parameters to control the behaviour of drainer
 type Helper struct {
-	Ctx                 context.Context
-	Client              kubernetes.Interface
-	Force               bool
-	GracePeriodSeconds  int
-	IgnoreAllDaemonSets bool
-	Timeout             time.Duration
-	DeleteLocalData     bool
-	Selector            string
-	PodSelector         string
-	Out                 io.Writer
-	ErrOut              io.Writer
+	Ctx                             context.Context
+	Client                          kubernetes.Interface
+	Force                           bool
+	GracePeriodSeconds              int
+	IgnoreAllDaemonSets             bool
+	Timeout                         time.Duration
+	DeleteLocalData                 bool
+	Selector                        string
+	PodSelector                     string
+	SkipWaitForDeleteTimeoutSeconds int
```

Comment:
I think we should add the KEP description of this feature as a comment here (the one that begins: "Alternative to "Unready Node Timeout" above...)

Reply:
Done.